### PR TITLE
Update background samples

### PR DESCRIPTION
**Pull request by abasnet97**
I have updated some CentralUL background samples and compared the old and new 2017 samples. The resulting plots seem consistent. Since I only considered two newly updated samples for 2017, the yields in CRs I considered were very small.

### CODE REVIEW

File: analysis/topEFT/make_cr_and_sr_plots.py [hunk 1]
```diff
@@ -435,7 +435,7 @@ def make_all_cr_plots(dict_of_hists,year,unit_norm_bool,save_dir_path):
     eft_sow_scale_dict = yt.get_eft_sow_scale_dict(dict_of_hists["SumOfEFTweights"])
 
     # Loop over hists and make plots
-    skip_lst = ["SumOfEFTweights"] # Skip this hist
+    skip_lst = ["SumOfEFTweights","ptz"] # Skip this hist
```
Comment (bryates) [inline analysis/topEFT/make_cr_and_sr_plots.py:438]: Do we want to skip `ptz`, or was this for a test?

File: analysis/topEFT/topeft.py [hunk 2]
```diff
@@ -348,8 +348,8 @@ def process(self, events):
                 # Prefiring weights only available in nanoAODv9**
                 weights_dict[ch_name].add('PreFiring', events.L1PreFiringWeight.Nom,  events.L1PreFiringWeight.Up,  events.L1PreFiringWeight.Dn)
                 # FSR/ISR weights
-                weights_dict[ch_name].add('ISR', events.ISRnom, events.ISRUp, events.ISRDown)
-                weights_dict[ch_name].add('FSR', events.FSRnom, events.FSRUp, events.FSRDown)
+                #weights_dict[ch_name].add('ISR', events.ISRnom, events.ISRUp, events.ISRDown)
+                #weights_dict[ch_name].add('FSR', events.FSRnom, events.FSRUp, events.FSRDown)
```
Comment (bryates) [inline analysis/topEFT/topeft.py:352]: Please uncomment

File: analysis/topEFT/work_queue_run.py [hunk 3]
```diff
@@ -166,8 +166,8 @@ def LoadJsonToSampleName(jsonFile, prefix):
 
 executor_args = {
     'master_name': '{}-workqueue-coffea'.format(os.environ['USER']),
-
-    'xrootdtimeout': 180,
+    'filepath': '/tmp', 
+    'xrootdtimeout': 300,
```
Comment (bryates) [inline analysis/topEFT/work_queue_run.py:170]: I know these are typical changes we've been using, but do we want to push them? @kmohrman what do you think?

File: analysis/topEFT/work_queue_run.py [hunk 4]
```diff
@@ -182,7 +182,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
     'schema': NanoAODSchema,
     'skipbadfiles': False,
 
-    'retries': 5,
+    'retries': 10,
```
Comment (bryates) [inline analysis/topEFT/work_queue_run.py:185]: Same as above

File: topcoffea/cfg/data_samples.cfg [hunk 5]
```diff
@@ -1,79 +1,79 @@
...
-root://ndcms.crc.nd.edu/
-../../topcoffea/json/data_samples/2016/DoubleEG_B-ver1_HIPM_UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_B-ver2_HIPM_UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_C-UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_D-UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_E-UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_F-HIPM_UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_F-UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_G-UL2016.json
-../../topcoffea/json/data_samples/2016/DoubleEG_H-UL2016.json
```
Comment (bryates) [inline topcoffea/cfg/data_samples.cfg:12]: Please restore all

File: topcoffea/cfg/data_samples_NDSkim.cfg [hunk 6]
```diff
@@ -1,79 +1,79 @@
...
-root://deepthought.crc.nd.edu/
-../../topcoffea/json/data_samples/2016/DoubleEG_B-ver1_HIPM_UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_B-ver2_HIPM_UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_C-UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_D-UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_E-UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_F-HIPM_UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_F-UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_G-UL2016_NDSkim.json
-../../topcoffea/json/data_samples/2016/DoubleEG_H-UL2016_NDSkim.json
```
Comment (bryates) [inline topcoffea/cfg/data_samples_NDSkim.cfg:12]: Revert changes in this file too

File: topcoffea/cfg/mc_signal_samples.cfg [hunk 7]
```diff
@@ -22,32 +22,32 @@ root://deepthought.crc.nd.edu/
...
-../../topcoffea/json/signal_samples/private_UL/UL18_ttHJet_b1.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttHJet_b2.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttHJet_b3.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttllJet_b1.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttllJet_b2.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttllJet_b3.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttlnuJet_b1.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttlnuJet_b2.json
-../../topcoffea/json/signal_samples/private_UL/UL18_ttlnuJet_b3.json
-../../topcoffea/json/signal_samples/private_UL/UL18_tttt_b4.json
```
Comment (bryates) [inline topcoffea/cfg/mc_signal_samples.cfg:41]: Restore

File: topcoffea/modules/corrections.py [hunk 8]
```diff
@@ -279,11 +279,11 @@ def AttachPSWeights(events):
   events['ISRnom']  = ak.ones_like(events.PSWeight[:,0])
   events['FSRnom']  = ak.ones_like(events.PSWeight[:,0])
   # Add up variation event weights
-  events['ISRUp']   = events.PSWeight[:, ISRup]
-  events['FSRUp']   = events.PSWeight[:, FSRup]
+  #events['ISRUp']   = events.PSWeight[:, ISRup]
+  #events['FSRUp']   = events.PSWeight[:, FSRup]
```
Comment (bryates) [inline topcoffea/modules/corrections.py:283]: Restore ISR/FSR

File: topcoffea/modules/corrections.py [hunk 9]
```diff
@@ -279,11 +279,11 @@ def AttachPSWeights(events):
...
   # Add down variation event weights
-  events['ISRDown'] = events.PSWeight[:, ISRdown]
-  events['FSRDown'] = events.PSWeight[:, FSRdown]
+  #events['ISRDown'] = events.PSWeight[:, ISRdown]
+  #events['FSRDown'] = events.PSWeight[:, FSRdown]
```
Comment (bryates) [inline topcoffea/modules/corrections.py:286]: Same here

**Review by bryates — CHANGES_REQUESTED** [review]
Overall, looks good. I just had a few comments on some things that were commented out. Maybe you didn't mean to push those?
Thanks for putting the check date on the ones that _still_ don't have `v9` versions.

File: analysis/topEFT/make_cr_and_sr_plots.py (hunk 1 above)
Comment (abasnet97) [reply analysis/topEFT/make_cr_and_sr_plots.py:438]: Oops. Sorry about that. I was having trouble making plots specifically for ptz CRs. Which is why I included it in that list.

File: analysis/topEFT/topeft.py (hunk 2 above)
Comment (abasnet97) [reply analysis/topEFT/topeft.py:352]: I will do that.

**Comment by abasnet97** [comment]
As @bryates pointed out, I mistakenly pushed some local changes to several files. I have reverted them back to their original form and pushed the new changes.

File: analysis/topEFT/work_queue_run.py [hunk 10]
```diff
@@ -166,7 +166,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
 
 executor_args = {
     'master_name': '{}-workqueue-coffea'.format(os.environ['USER']),
-
+
```
Comment (kmohrman) [inline analysis/topEFT/work_queue_run.py:169]: This is a very minor comment, but it'd be good to not add this extraneous white space.

Comment (abasnet97) [reply analysis/topEFT/work_queue_run.py:169]: I will get rid of that.

File: topcoffea/cfg/data_samples_NDSkim.cfg [hunk 11]
```diff
@@ -58,7 +58,7 @@ root://deepthought.crc.nd.edu/
 ../../topcoffea/json/data_samples/2017/DoubleMuon_F-UL2017_NDSkim.json
 
 ../../topcoffea/json/data_samples/2017/MuonEG_B-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/MuonEG_C-UL2017_NDSkim.json
+../../topcoffea/json/data_samples/2017/MuonEG_C-UL2017_NDSkim.json#
```
Comment (kmohrman) [inline topcoffea/cfg/data_samples_NDSkim.cfg:61]: Any reason for this trailing `#`? If not then maybe we can remove it.

Comment (abasnet97) [reply topcoffea/cfg/data_samples_NDSkim.cfg:61]: removed

**Review by bryates — APPROVED** [review]
Thanks for the fixes @abasnet97!

File: analysis/topEFT/work_queue_run.py [hunk 12]
```diff
@@ -165,8 +165,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
 processor_instance = topeft.AnalysisProcessor(samplesdict,wc_lst,hist_lst,do_errors,do_systs,split_lep_flavor,skip_sr,skip_cr)
 
 executor_args = {
-    'master_name': '{}-workqueue-coffea'.format(os.environ['USER']),
-
```
[the remark this answers is not in the document]
Comment (abasnet97) [reply analysis/topEFT/work_queue_run.py]: This should be fixed now.

**Comment by kmohrman** [comment]
Thanks @abasnet97! This looks good to me!

**Comment by kmohrman** [comment]
Looks like the CI has passed and @bryates has also approved it as well, so I'd be happy to merge it now.

**Pull request merged**